src/components/forms/SshKeyForm.js

```diff
@@ -43,7 +43,8 @@ class SshKeyForm extends Component {
       <>
         <IcseFormGroup>
           {/* use data */}
-          <IcseToggle
+          {!this.state.power_vs && (
```

Comment:
should be `this.props.power_vs`, should not be passed as part of data

idea here is that this parameter will be added when adding an ssh key subform for power vs resources